### ISSUE TITLE
sidebar slide animation

### ISSUE THREAD

**Issue by JeffWScott** [halfmoonui/halfmoon#77]
The sidebar appears and disappears when toggled. But it doesn't have the nifly slide animation like on the main halfmoon site.

Is this something I need to roll myself or could it be a configuration error on my part?

**Comment by halfmoonui** [comment]
Did you include `halfmoon.js` in your project? Right now, there is a small issue where the `.with-transitions` class is attached to the page wrapper using JavaScript. This class actually provides the transition you are looking for.

This was done to avoid a really annoying bug on Chrome that runs transitions on page load. I am looking for ways to change this in a future update.

**Comment by JeffWScott** [comment]
Yeah I have the JavaScript file via CDN. All the functionality of it seems to work fine.

Now that I know that animation is supposed to work I can dig in and figure out why.
